.codecov.yml

```diff
@@ -1,17 +1,14 @@
 # Documentation: https://docs.codecov.io/docs/codecov-yaml
 
-codecov:
-  bot: dlang-bot
-
```

Comment:
Yes eventually they will, but by default they will wait for all CIs. Those four lines will trigger the notification on the first submission - don't ask me why they don't send it by default.